specification/devcenter/DevCenter/DevBox/routes.tsp

```diff
@@ -6,22 +6,80 @@ import "../shared/routes.tsp";
 using Azure.Core;
 using TypeSpec.Rest;
 using TypeSpec.Http;
+using TypeSpec.Versioning;
 
 namespace DevCenterService;
 
 interface DevBoxes {
-  @doc("Lists available pools")
+  @doc("Lists available pools.")
   listPools is StandardResourceOperations.ResourceList<Pool>;
 
-  @doc("Gets a pool")
+  @doc("Gets a pool.")
   getPool is StandardResourceOperations.ResourceRead<Pool>;
 
-  @doc("Lists available schedules for a pool.")
-  listSchedulesByPool is StandardResourceOperations.ResourceList<Schedule>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Does not fit any standard operation pattern since DevBox has a different path"
+  @doc("Lists Dev Boxes that the caller has access to in the DevCenter.")
+  @route("/devboxes")
+  @get
+  listAllDevBoxes is Azure.Core.Foundations.Operation<{}, DevBoxListResult>;
```

Comment:
I recently found that you could use the Foundation.ResourceList operation in this case. It might at least bring you closer to the standard resource list operation. You'll still leave the suppression though.

Reply:
Ah I see, ok then it's fine as is.